src/create_wand_data.cpp

```diff
@@ -41,6 +43,7 @@ int main(int argc, const char **argv)
     app.add_flag("--compress", compress, "Compress additional data");
     app.add_option("-s,--scorer", scorer_name, "Scorer function")->required();
     app.add_flag("--range", range, "Create docid-range based data")->excludes(var_block_opt);
+    app.add_option("--dropped_terms", dropped_terms_filename, "A filename containing a list of term IDs that we want to drop");
```

Comment:
Why not `--terms_to_drop`? In same way, the description maybe could be improved as:` "A filename containing a list of term IDs to drop"`.

Reply:
I actually agree, "terms to drop" is way clearer. But I would personally go with `--terms-to-drop` with dashes instead of underscores.